front/app/containers/PlatformFooter/index.tsx

```diff
@@ -345,7 +345,11 @@ const PlatformFooter = ({ className }: Props) => {
                 <FormattedMessage {...messages.poweredBy} />
               </PoweredByText>
               <GoVocalLink href="https://govocal.com/" target="_blank">
-                <GoVocalLogo name={getLocalizedLogoName()} title="Go Vocal" />
+                <GoVocalLogo
+                  ariaHidden={false}
+                  name={getLocalizedLogoName()}
+                  title="Go Vocal"
+                />
```

Comment:
@alexander-cit Do we need an alt tag or aria label here? Or does the name cover that?

Reply:
Good question! We recently started to use `title` for SVG Icons https://github.com/CitizenLabDotCo/citizenlab/pull/8061/files#diff-10700d79d71c0bcfe457acb8744ecbb04bee0113b660996d6a626eed51abeb50

Reply:
Ah gotchya! Didn't know that. Thanks Edwin! :+1: